backend/tests/apps/core/api/status_test.py

```
@@ -0,0 +1,32 @@
+    """Test suite for the status endpoint."""
+
+    @pytest.fixture(autouse=True)
+    def setup_client(self):
+        self.client = Client()
+        self.url = reverse("get_status")
+
+    def test_status_endpoint_valid_request(self):
+        """Test a valid GET request to the status endpoint."""
+        response = self.client.get(self.url)
+
+        assert response.status_code == codes.ok
```

Pylint (3.3.7)

[error] Instance of 'LookupDict' has no 'ok' member

(E1101)